templates_test/insert.tpl

```diff
@@ -41,7 +41,11 @@ func test{{$tableNamePlural}}InsertWhitelist(t *testing.T) {
 
 	tx := MustTx(boil.Begin())
 	defer tx.Rollback()
+	{{if ne .DriverName "mssql"}}
 	if err = {{$varNameSingular}}.Insert(tx, {{$varNameSingular}}Columns...); err != nil {
+	{{- else -}}
+	if err = {{$varNameSingular}}.Insert(tx, {{$varNameSingular}}ColumnsWithoutDefault...); err != nil {
```

Comment:
In my previous comment I do say we should get rid of all special casing, but this particular change actually weakens the test's purpose so we should leave this alone.